shell/platform/common/accessibility_bridge.h

```diff
@@ -63,6 +63,7 @@ class AccessibilityBridge
   class AccessibilityBridgeDelegate {
    public:
     virtual ~AccessibilityBridgeDelegate() = default;
+
```

Comment:
What do you think of updating `AccessibilityBridge`'s comments to mention there is one bridge per view?

Reply:
Good point. Although I couldn't find much more to add, but to say 
```
/// Use this class to maintain the accessibility tree for a view. (...)
```

What do you think?

Reply:
Your new comment looks perfect 👍